src/App.jsx

```diff
@@ -14,33 +15,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goodsToSort, setGoodsToSort] = useState(goodsFromServer);
+  const [selectedOption, setSelectedOption] = useState("");
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const optionsToSort = [
+    { type: "Sort alphabetically", className: "is-info", isToAppear: true },
+    { type: "Sort by length", className: "is-success", isToAppear: true },
+    { type: "Reverse", className: "is-warning", isToAppear: true },
+    { type: "Reset", className: "is-danger", isToAppear: !checkIfArrayIsEqualsToFromServer() }
+  ];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  function handleGoodsAlphabetically() {
+    if(selectedOption === "Reverse") {
+    setGoodsToSort([...goodsToSort].sort((a, b) => a.localeCompare(b)).reverse())
```

Comment:
In the `handleGoodsAlphabetically` function, setting `selectedOption` to 'Sort alphabetically' even when 'Reverse' is selected is misleading. You should set `selectedOption` to 'Reverse' in this case to reflect the actual operation performed.